src/webviews/apps/settings/partials/commit-graph.html

```diff
@@ -170,6 +170,20 @@ <h2>
 						</div>
 					</div>
 
+					<div class="setting">
```

Comment:
Can do it outside this PR, but we should look at the UX/UI of the graph settings and group some of them up to make them easier for users to understand

Reply:
Wrote up a task for it, thanks!